api/src/pcapi/settings.py

```diff
@@ -150,6 +150,8 @@
 ALGOLIA_LAST_30_DAYS_BOOKINGS_RANGE_THRESHOLDS = [
     int(value) for value in secrets_utils.get("ALGOLIA_LAST_30_DAYS_BOOKINGS_RANGE_THRESHOLDS", "1,2,3,4").split(",")
 ]
+ENABLE_UNINDEXING_ALL = bool(int(os.environ.get("ENABLE_UNINDEXING_ALL", 1)))
```

Comment:
Pour ce genre de setting dangereux, je suggère de **dés**activer par défaut : 

```suggestion
ENABLE_UNINDEXING_ALL = bool(int(os.environ.get("ENABLE_UNINDEXING_ALL", 0)))
```

Ainsi, si jamais on a une typo dans le nom de la variable d'environnement, ou je ne sais quel autre bug qui fait qu'on se retrouve avec la valeur par défaut, on ne risque pas d'activer cette option à tort sur la prod.